doc/release_notes_0.6.rst

```diff
@@ -0,0 +1,196 @@
+.. _release_notes_0.6:
+
+ACRN v0.6 (Feb 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.6.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, and documentation.
+You can either download this source code as a zip or tar.gz file (see
+the `ACRN v0.6 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.6>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.6
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.6 documents can be found at
+https://projectacrn.github.io/0.6/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+ACRN v0.6 requires Clear Linux OS version 27600 or newer.  Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.6 new features
+************************
+
+**Support PREEMPT_RT-Linux for real-time UOS in ACRN**:
```

Comment:
I would like that we more clearly split the difference between the ACRN ability to run "Privileged VMs" and the "recipe" we have published to use a UOS kernel that includes the PREEMPT_RT patches. Both are linked but should be loosely coupled. In my opinion, this paragraph should emphasize the fact we are enabling "Privileged VMs" in ACRN, this is what the title should reflect. The paragraph can also mention that as a result, we have published a tutorial on how to also use the PREEMPT_RT patches in a UOS kernel.